codes/models/projection_helper_functions.R

```diff
@@ -288,4 +257,62 @@ compute_predicted_cum_cases <- function(full_data, model, policy_variables_used,
       by = "date"
     )
   out  
-}
\ No newline at end of file
+}
+
+# Here we start at cum_confirmed_cases[1] - predict itself for the first one (exp(0))
+# Then predict the using the sum of the log changes from the second on
+calculate_projection_for_one_unit <- function(cum_confirmed_cases_first, 
+                                              prediction_logdiff,
+                                              time_steps_per_day,
+                                              daily_gamma,
+                                              unit_population,
+                                              proportion_confirmed){
+  # cum_confirmed_cases == I + R
+  # number_of_infectious_individuals == I
+  stopifnot(is.na(prediction_logdiff[1]))
+  stopifnot(!is.na(cum_confirmed_cases_first))
+  
+  # the simulation will start on the first hour of the second day
+  # the seed of the simulation is the last hour of the first day - which we assume to be 
+  # the number from the data
+  
+  cum_confirmed_cases_simulated <- rep(NA_real_, (length(prediction_logdiff) - 1)*time_steps_per_day + 1)
+  number_of_infectious_individuals <- cum_confirmed_cases_simulated # NA
+  number_of_susceptible_individuals <- cum_confirmed_cases_simulated # NA
+  number_of_recovered_individuals <- cum_confirmed_cases_simulated # NA
+  
+  # Assumption here is that all individuals are infectious initially
+  number_of_infectious_individuals[1] <- cum_confirmed_cases_first/proportion_confirmed
+  
+  cum_confirmed_cases_simulated[1] <- cum_confirmed_cases_first
+  number_of_susceptible_individuals[1] <- unit_population - number_of_infectious_individuals[1]
+  number_of_recovered_individuals[1] <- 0
+  
+  prediction_logdiff_interpolated <- c(NA_real_, rep(prediction_logdiff[-1], each = time_steps_per_day))
+  stopifnot(length(prediction_logdiff_interpolated) == length(cum_confirmed_cases_simulated))
+  new_gamma = (1 + daily_gamma)^(1/time_steps_per_day) - 1
+  for(i in 2:length(cum_confirmed_cases_simulated)){ # adm1_pop is constant here
+    # new_infections <- number_of_infectious_individuals[i - 1]*
+    #   exp((prediction_logdiff[i] + gamma)*max(population[1] - out[i - 1], 0)/population[1]) -
+    #   number_of_infectious_individuals[i - 1]
+    
+    # i_t+1 = i_t exp(beta*S - gamma)
+    number_of_infectious_individuals[i] = 
+      number_of_infectious_individuals[i - 1]*exp((prediction_logdiff_interpolated[i]/time_steps_per_day + new_gamma)*
+                                                    number_of_susceptible_individuals[i - 1]/unit_population - 
+                                                    new_gamma)
+     
+    recoveries <- number_of_infectious_individuals[i - 1]*(exp(new_gamma) - 1)
```

Comment:
@kendonB I think this should technically be:
`recoveries <- number_of_infectious_individuals[i - 1] * new_gamma`

for small `new_gamma`,  `exp(new_gamma)-1`  is obviously very close to `new_gamma` so it should in general be fine.  But if we pass large-ish values of `gamma` with six daily time-steps, it is possible this could be a little biased.  Also, it might be easier for a reader to follow this way, as it directly replicates Eq. 10 in the paper.

Reply:
It's not obvious to me which of these is the best discrete approximation - as you point out both are correct in the limit. One will converge faster than the other but I don't know which.

Reply:
It just felt like the exponential would be closer to me but I'm willing to be convinced

Reply:
I don't feel strongly about this, and it definitely does not matter at all for small `new_gamma`.   Happy to leave it as it is.  I trust your instincts on convergence in this situation.